src/Overlay/GlobalOverlay/GlobalOverlay.tsx

```diff
@@ -60,7 +83,9 @@ export const GlobalOverlay: FC<GlobalOverlayProps> = ({
                     className={backdropClassName}
                   />
                 )}
-                {children({ overlayIndex, portalIndex })}
+                <div body-scroll-lock-ignore="true">
```

Comment:
Do we always want this?

Reply:
Yea I think so. `disableScroll` above disables touchmove events and then `body-scroll-lock-ignore` ensures the touchmove events don't get disabled for the children of this div

Reply:
something like `body-scroll-touchmove-enable` would probably be a more clear name cuz it only has an effect on touchmove events. I named it the same as in the body-scroll-lock-upgrade docs example